charts/kserve-llmisvc-crd-minimal/README.md

```
@@ -2,12 +2,12 @@
 Helm chart for deploying LLMInferenceService minimal crds
 
-![Version: v0.20.0-rc0](https://img.shields.io/badge/Version-v0.20.0--rc0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.20.0-rc0](https://img.shields.io/badge/AppVersion-v0.20.0--rc0-informational?style=flat-square)
+![Version: v0.20.0](https://img.shields.io/badge/Version-v0.20.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.20.0](https://img.shields.io/badge/AppVersion-v0.20.0-informational?style=flat-square)
 
 ## Installing the Chart
 
 To install the chart, run the following:
 
 ```console
-$ helm install kserve-llmisvc-crd-minimal oci://ghcr.io/kserve/charts/kserve-llmisvc-crd-minimal --version v0.20.0-rc0
+$ helm install kserve-llmisvc-crd-minimal oci://ghcr.io/kserve/charts/kserve-llmisvc-crd-minimal --version v0.20.0
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)